### DISCUSSION TITLE
Adapter no Lazarus

### DISCUSSION THREAD

**Discussion by josinaldobezerra** [viniciussanchez/RESTRequest4Delphi#233] [General]
Olá,
como posso utilizar o adapter no Lazarus ?

**Comment by viniciussanchez** [comment]
Acredito que da mesma forma como no Delphi... ou está tendo algum problema? Faz um tempo que nao uso Lazarus

**Comment by josinaldobezerra** [comment]
Também não uso muito Lazarus, mas peguei uma situação em um projeto antigo que seria interessante usar o componente com o adapter, mas, desculpe minha ignorancia, mas não tem memtable no lazarus. Ou pode utilizar outro tipo de componente para utilzar como dataset ?

Acabei utilizando o restrequest4d para consulta mas o json tratei na mão mesmo, sem adpater. 

Mas fiquei curioso pra saber se tem como e com qual componente.

**Reply by viniciussanchez** [reply]
Sim, realmente não tem o FireDAC (TFDMemTable), mas pode usar qualquer componente que tenha como herança um TDataSet. Por exemplo Zeos... UniDAC, etc...

**Reply by josinaldobezerra** [reply]
Ok, obrigado.
